addons/crm_claim/report/crm_claim_report.py

```diff
@@ -1,50 +1,39 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp.osv import fields,osv
-from openerp import tools
+from odoo import fields, models, tools
 
-AVAILABLE_PRIORITIES = [
-   ('0', 'Low'),
-   ('1', 'Normal'),
-   ('2', 'High')
-]
 
-
-class crm_claim_report(osv.osv):
+class CrmClaimReport(models.Model):
     """ CRM Claim Report"""
 
     _name = "crm.claim.report"
-    _auto = False
     _description = "CRM Claim Report"
+    _auto = False
 
-    _columns = {
-        'user_id':fields.many2one('res.users', 'User', readonly=True),
-        'team_id':fields.many2one('crm.team', 'Team', oldname='section_id', readonly=True),
-        'nbr': fields.integer('# of Claims', readonly=True),  # TDE FIXME master: rename into nbr_claims
-        'company_id': fields.many2one('res.company', 'Company', readonly=True),
-        'create_date': fields.datetime('Create Date', readonly=True, select=True),
-        'claim_date': fields.datetime('Claim Date', readonly=True),
-        'delay_close': fields.float('Delay to close', digits=(16,2),readonly=True, group_operator="avg",help="Number of Days to close the case"),
-        'stage_id': fields.many2one ('crm.claim.stage', 'Stage', readonly=True,domain="[('team_ids','=',team_id)]"),
-        'categ_id': fields.many2one('crm.claim.category', 'Category',readonly=True),
-        'partner_id': fields.many2one('res.partner', 'Partner', readonly=True),
-        'company_id': fields.many2one('res.company', 'Company', readonly=True),
-        'priority': fields.selection(AVAILABLE_PRIORITIES, 'Priority'),
-        'type_action': fields.selection([('correction','Corrective Action'),('prevention','Preventive Action')], 'Action Type'),
-        'date_closed': fields.datetime('Close Date', readonly=True, select=True),
-        'date_deadline': fields.date('Deadline', readonly=True, select=True),
-        'delay_expected': fields.float('Overpassed Deadline',digits=(16,2),readonly=True, group_operator="avg"),
-        'email': fields.integer('# Emails', size=128, readonly=True),
-        'subject': fields.char('Claim Subject', readonly=True)
-    }
+    user_id = fields.Many2one('res.users', string='User', readonly=True)
+    team_id = fields.Many2one('crm.team', string='Team', oldname='section_id', readonly=True)
+    nbr = fields.Integer(string='# of Claims', readonly=True)  # TDE FIXME master: rename into nbr_claims
```

Comment:
FIXME ? I think we should fix it as we are migrating and refactoring code, we should change the field name with oldname